.circleci/config.yml

```
@@ -20,31 +20,45 @@ e2eTest: &e2eTest
       command: ./e2e-harness teardown
       when: always
 
+orbs:
+  architect: giantswarm/architect@0.6.0
 
-
-version: 2
+version: 2.1
 jobs:
   build:
     machine: true
     steps:
-    - checkout
-    - run:
-        name: Install architect
-        command: |
+      - checkout
+
+      - run: |
```

Build step name missing, e.g.:
```suggestion
      - run:
          name: Install architect
          command: |
            wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
            chmod +x ./architect
            ./architect version
```